docs/dapp/opl/README.md

```diff
@@ -11,13 +11,60 @@ The Oasis Privacy Layer contains a set of [smart contracts](https://github.com/o
 that integrate [Sapphire](../sapphire/README.mdx) into your existing and future
 Web3 applications, introducing confidentiality.
 
-Application developers on any EVM-compatible network can add confidential state
-and selective disclosures to their application through the usage of a smart
-contract deployed on OPL without migrating their application stack. OPL enabled
-smart contracts provide a default bridge for developers to pass messages
-between Sapphire and other networks.
+Application developers on [any supported network](https://im-docs.celer.network/developer/contract-addresses-and-rpc-info) can add confidential state
```

Comment:
I have added a list (condensed) which enumerates the mainnets & testnets. If this needs expanding into a table it will need to be shifted down the page as it will take up significant vertical space.